draft-ietf-quic-transport.md

```diff
@@ -706,9 +706,14 @@ frames count toward flow control.
 
 A STOP_SENDING frame requests that the receiving endpoint send a RST_STREAM
 frame.  An endpoint that receives a STOP_SENDING frame MUST send a RST_STREAM
-frame for that stream, and can use an error code of STOPPING.  If the
-STOP_SENDING frame is received on a send stream that is already in the "Data
-Sent" state, a RST_STREAM frame MAY still be sent in order to cancel
+frame for that stream.  An endpoint can use any application error code, though
+they SHOULD send the error code with a value of 0.  The endpoint that sends a
```

Comment:
I think there could be non-fatal errors resulting in STOP_SENDING so “should send 0” is not that helpful.

For example, error could be - upstream bucket is full, or logging feedback disabled on channel.